apps/Cargo.toml

```diff
@@ -33,9 +33,9 @@ gst-rtsp-server = { package = "gstreamer-rtsp-server", git = "https://gitlab.fre
 gst-sdp = { package = "gstreamer-sdp", git = "https://gitlab.freedesktop.org/gstreamer/gstreamer-rs" }
 gstreamer-video = { git = "https://gitlab.freedesktop.org/gstreamer/gstreamer-rs" }
 gtk = { git = "https://github.com/gtk-rs/gtk-rs" }
-pravega-client = { git = "https://github.com/pravega/pravega-client-rust", rev = "9ed43aaefabfb2750a2361419a7232200e6ed4a0" }
-pravega-client-config = { git = "https://github.com/pravega/pravega-client-rust", package = "pravega-client-config", rev = "9ed43aaefabfb2750a2361419a7232200e6ed4a0" }
-pravega-client-shared = { git = "https://github.com/pravega/pravega-client-rust", package = "pravega-client-shared", rev = "9ed43aaefabfb2750a2361419a7232200e6ed4a0" }
+pravega-client = { git = "https://github.com/pravega/pravega-client-rust", rev = "6614df550bd286e3d1867022c3f235836c6cc278" }
```

Comment:
How are these changes related to the healthcheck?

Reply:
This PR was based on the previous PR. Only the last commit was specifically related to this PR.

Reply:
Do you mean that we needed to update pravega-client to get the healthcheck working properly?